client/metric_conditions.go

```diff
@@ -119,6 +119,7 @@ func (c *Client) CreateMetricCondition(
 			Expression:    condition.Attributes.Expression,
 			Queries:       condition.Attributes.Queries,
 			AlertingRules: condition.Attributes.AlertingRules,
+			Description:   condition.Attributes.Description,
```

Comment:
🔥 looks like the [acceptance test](https://github.com/lightstep/terraform-provider-lightstep/blob/17f8d52e5a23b9099dc3eaaaf5d3cab5d2189fb2/lightstep/resource_metric_condition_test.go) never exercised this bit; mind updating those test cases to test descriptions?

Reply:
I added a description to a create and update! it looks like all the acceptance tests are failing though, maybe with a permissions issue.. any ideas as to why?

Reply:
hmmm my gut says [this dude](https://github.com/lightstep/terraform-provider-lightstep/blob/main/.github/workflows/pr.yml#L109) expired, but i lack permissions to update it 😬 i think we have to find someone powerful enough to update this secret

Reply:
done!